app/models/keyword.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Keyword < ApplicationRecord
+  belongs_to :user
+
+  enum status: { in_progress: 0, completed: 1, failed: 2 }
+
+  def decorated
```

Comment:
@suho This will initialize a new object everytime, better to hoist it. 
```
   def decorator
     @decorator ||= KeywordDecorator.new(self)
   end
```

Reply:
Fixed in [`f42675d` (#40)](https://github.com/suho/google-search-ruby/pull/40/commits/f42675dd1e9b7fdfe0b06e96e5e8709d750dd695)